schema.json

```
@@ -459,77 +490,99 @@
             ]
         },
         "orcid": {
+            "description": "Identifier for an author, see https://orcid.org.",
             "format": "uri",
             "pattern": "https://orcid\\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}",
             "type": "string"
         },
         "person": {
+            "description": "A person.",
             "type": "object",
             "additionalProperties": false,
             "properties": {
                 "address": {
-                    "$ref": "#/definitions/address"
+                    "$ref": "#/definitions/address",
+                    "description": "The person's address."
                 },
                 "affiliation": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The person's affilitation."
                 },
                 "alias": {
-                    "$ref": "#/definitions/alias"
+                    "$ref": "#/definitions/alias",
+                    "description": "The person's alias."
                 },
                 "city": {
-                    "$ref": "#/definitions/city"
+                    "$ref": "#/definitions/city",
+                    "description": "The person's city."
                 },
                 "country": {
-                    "$ref": "#/definitions/country"
+                    "$ref": "#/definitions/country",
+                    "description": "The person's country."
                 },
                 "email": {
-                    "$ref": "#/definitions/email"
+                    "$ref": "#/definitions/email",
+                    "description": "The person's email address."
                 },
                 "family-names": {
+                    "description": "The person's family names.",
                     "type": "string"
                 },
                 "fax": {
-                    "$ref": "#/definitions/fax"
+                    "$ref": "#/definitions/fax",
+                    "description": "The person's fax number."
                 },
                 "given-names": {
+                    "description": "The person's given names.",
                     "type": "string"
                 },
                 "name-particle": {
+                    "description": "The person's name-particle.",
```

```suggestion
                    "description": "The person's name particle, e.g., a nobiliary particle or a preposition meaning of or from.",
                    "examples": [
                        "'von' (for Alexander von Humboldt)"
                    ],
```

Add example and extend description for those unfamiliar with nobiliary particles, etc.

---

1. not sure the description is correct right now?
2. `examples` is meant to (only) show examplary use of a key, not including a description of the case.

---

Thanks. 

1. Isn't it? not even with the `e.g.` in there?
2. I see. How about this:

```suggestion
                    "description": "The person's name particle, e.g., a nobiliary particle or a preposition meaning 'of' or 'from' (for example 'von' in 'Alexander von Humboldt').",
                    "examples": [
                        "von"
                    ],
```

---

Yes! I see what you mean now